ui_framework/src/components/typography/typography.js

```diff
@@ -4,15 +4,30 @@ import {
 } from 'react';
 import classNames from 'classnames';
 
-const sizeToClassNameMap = {
+const titleSizeToClassNameMap = {
   small: 'kuiTitle--small',
   large: 'kuiTitle--large',
 };
 
-export const SIZES = Object.keys(sizeToClassNameMap);
+const titleSizeToVerticalRhythmClassNameMap = {
+  small: 'kuiVerticalRhythm',
+  large: 'kuiVerticalRhythm--xLarge',
```

Comment:
Agree. I don't have a solution, but think this is the best way to "solve" it. Basically that component sizing and CSS global sizing are somewhat distanced to make React components more usable.